packages/core/src/index.ts

```diff
@@ -6,3 +6,4 @@ export { Ceramic } from './ceramic.js'
 export * from './store/ikv-store.js'
 export * from './ceramic.js'
 export * from './pubsub/pubsub-message.js'
+export { InMemoryAnchorService } from './anchor/memory/in-memory-anchor-service.js'
```

Comment:
Why?

Reply:
the anchor checking code needs to cast to this type for it to work. Rather than having core functionality in common and hoping the cast works, now it's explicit. But it did take an export.